### ISSUE TITLE
Suggestions? Post here or somewhere else?

### ISSUE THREAD

**Issue by cdmichaelb** [ensingm2/SteamMonsterGameScript#106]
Suggestions aren't technically an issue but I don't see anywhere else to post them. Should they be posted in the issues section?

**Comment by vanZeben** [comment]
Yes. There is an enhancement tag for suggestions

**Comment by ensingm2** [comment]
^ What he said. I'll leave this 'Issue' open in case anyone else stumbles on it with the same question.

**Comment by cdmichaelb** [comment]
Is that tag something we add on somehow, or does a contributor see it and tag it?

**Comment by ensingm2** [comment]
After creating the issue, look to the right of the comments and apply a label

**Comment by thelulzy** [comment]
I don't believe we have the rights to apply labels. I do not seem to be able to apply one to my newest question.

**Comment by ensingm2** [comment]
Hmmm... Oh well, just post an issue, not too difficult to figure out what's a bug vs a suggestion.
